File: CatalogService/src/main/java/ecom/catalog/CatalogService/POJO/Product.java
package ecom.catalog.CatalogService.POJO;

import java.util.*;

import jakarta.persistence.*;

@Entity
@Table(name = "products")
public class Product {

    @Id
    private UUID productGuid;
    private String name;
    @Column(length = 4000)
    private String description;
    private String dimensions;
    private double weight;
    private double price;
    private int amount;

    public UUID getProductGuid() {
        return productGuid;
    }

    public void setProductGuid(UUID productGuid) {
        this.productGuid = productGuid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDimensions() {
        return dimensions;
    }

    public void setDimensions(String dimensions) {
        this.dimensions = dimensions;
    }

    public double getWeight() {
        return weight;
    }

    public void setWeight(double weight) {
        this.weight = weight;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }
}

File: CheckOutService/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace CheckOutService.Models
{
    public class Product
    {
        [Key]
        public Guid productGuid { get; set; }

        [Required]
        public Guid orderGuid { get; set; }

        [Required] 
        public Order order { get; set; }

        [Required]
        public string name { get; set; }

        [Required]
        [MaxLength(4000)]
        public string description { get; set; }

        [Required]
        public string dimensions { get; set; }

        [Required]
        public double weight { get; set; }

        [Required]
        public double price { get; set; }

        [Required]
        public int amount { get; set; }
    }
}
